Dockerfile

```diff
@@ -6,11 +6,14 @@ ARG GITHUB_TOKEN
 RUN apt update && apt install -y git procps
 RUN pip install --upgrade pip
 
+RUN pip install jupyter
+
+RUN echo 'alias la="ls -la"' >> ~/.bashrc
+RUN echo 'alias jn="jupyter notebook --ip 0.0.0.0 --allow-root --no-browser --port=8889"' >> ~/.bashrc
```

Comment:
Are these two commands used?

Reply:
yes I do, as I run notebooks in this container. but skip if you do not deem it usefil